content/en/docs/external-add-ons/kserve/first_isvc_kserve.md

```diff
@@ -0,0 +1,196 @@
++++
+title = "Run your first InferenceService"
+description = "None"
```

Comment:
You want to leave this as empty string other it shows up as "None" on the page.

Reply:
It was autogenerated from the notebook so it will be none

Reply:
Learned the notebook needed a quote containing the description so that the generation script will correctly add the description to the markdown file.  I went ahead and added a description and regenerated.